src/Tizen.NUI/src/internal/Common/RenderTask.cs

```diff
@@ -48,9 +48,12 @@ protected override void Dispose(DisposeTypes type)
                 return;
             }
 
-            foreach (var window in Application.GetWindowList())
```

Comment:
How about using this? '?? Enumerable.Empty<string>()'

```csharp
foreach (var window in Application.GetWindowList() ?? Enumerable.Empty<string>())
```

Reply:
Done.
Thanks.